cmd/keycloakb/keycloak_bridge.go

```diff
@@ -831,7 +841,11 @@ func config(logger log.Logger) *viper.Viper {
 	sort.Strings(keys)
 
 	for _, k := range keys {
-		logger.Log(k, v.Get(k))
+		if _, censored := censoredParameters[k]; censored {
```

Comment:
I would add another condition : if "password" is found in the parameter name, it is automatically censored (and perhaps some other keywords)

Reply:
Added

Reply:
With your latest fix, you don't need anymore rows like `censoredParameters["XXX-password"] = true`

Reply:
Another idea: only 2 branches for if statement with 

> censored || strings.Contains(k, "password")